src/pages/MS/Login/LoginMS.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
+import "./LoginMS.scss";
+
+class LoginMS extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: "",
+      pw: "",
+      active: false,
+    };
+  }
+  handleIdInput = e => {
+    console.log(e.target.value);
+    this.setState = { id: e.target.value };
+    console.log(e.target.value.includes("@") && this.state.pw.length > 5);
+    if (e.target.value.includes("@") && this.state.pw.length > 5) {
+      this.setState = { active: true };
+    } else {
+      this.setState = { active: false };
+    }
+  };
```

Comment:
```suggestion
    this.setState = { active: e.target.value.includes("@") && this.state.pw.length > 5 };
```

validation 하는 로직 자체가 true 혹은 false가 됩니다.

Reply:
수정하겠습니다!